src/Serilog/LoggerConfiguration.cs

```diff
@@ -19,14 +19,15 @@ namespace Serilog;
 /// </summary>
 public class LoggerConfiguration
 {
-    readonly List<ILogEventSink> _logEventSinks = [];
```

Comment:
I'd prefer not to make members in `LoggerConfiguration` lazily initialized - clarity is important in this class, and the savings seem like they will be modest.